src/live-stream.js

```diff
@@ -675,15 +675,18 @@ class LiveStream {
   }
 
   readyOGVInstance() {
-   // if  (!this.ogvPlayer ) {
+    let container;
+    if  (!this.ogvPlayer ) {
       const playerOptions = { forceWebGL: true, debug: false};
       this.ogvPlayer = new OGVPlayer(playerOptions);
-      var container = document.createElement('div');
-      container.id = 'ogvVideoContainer';
-      container.appendChild(this.ogvPlayer);
-      document.getElementById('liveStreamCont').appendChild(container);
+      
       // container.style.position = 'absolute';
-    // }
+     }
```

Comment:
Codacy found an issue: [Expected indentation of 8 spaces but found 6.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=6073643)